{{cookiecutter.project_slug}}/pyproject.toml

```diff
@@ -0,0 +1,101 @@
+[project]
+name = "{{cookiecutter.project_slug}}"
+version = "{{cookiecutter.project_version}}"
+dependencies = [
+    "Django<3.3",
+    "bcrypt",
+    "django-behaviors",
+    "django-environ",
+    "django-ipware",
+    "django-axes",
+    "whitenoise",
+    "django-storages",
+    "djangorestframework",
+    "djangorestframework-camel-case",
+    "drf-jwt",
+    "drf-spectacular[sidecar]",
+    "django-filter",
+    "django-split-settings",
+    "django-healthchecks",
+    "redis",
+    "sentry-sdk",
+    "Pillow",
+]
+
+
+[project.optional-dependencies]
+dev = [
+    "ipython",
+
+    "pytest-django>=3.9",
+    "pytest-deadfixtures",
+    "pytest-env",
+    "pytest-freezegun",
+    "pytest-mock",
+    "pytest-randomly",
+
+    "dotenv-linter",
+
+    "freezegun",
+    "mixer",
+
+    "jedi",
+    "flake8-absolute-import",
+    "flake8-aaa",
+    "flake8-bugbear",
+    "flake8-cognitive-complexity",
+    "flake8-commas",
+    "flake8-django",
+    "flake8-eradicate",
+    "flake8-isort>=4.0.0",
+    "flake8-fixme",
+    "flake8-multiline-containers",
+    "flake8-mutable",
```

Comment:
flake8-bugbear has already  `B006 ` for checking for mutable default argument so that plugin can be removed.